controllers/user.js

```diff
@@ -57,11 +57,13 @@ exports.postLogin = (req, res, next) => {
  */
 exports.logout = (req, res) => {
   req.logout();
-  req.session.destroy((err) => {
+  /* req.session.destroy((err) => {
```

Comment:
OWASP requirement for logout includes invalidating the user session: https://www.owasp.org/index.php/Logout

Reply:
Sir, can you explain it. I am not able to understand.

Reply:
See the link provided.